resume.page

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Meryl Streep- Top 10</title>
+  <link rel="stylesheet" href="stylesheet.css" type="text/css" media="screen">
+  <nav>
+    <a href="index.html">Bio |</a>
+    <a href="resume.page">Greatest Hits |</a>
+    <a href="http://www.imdb.com/name/nm0000658/">More Info |</a>
+  </nav>
+</head>
+<body background="chinon-super8-moviecamera-6455277-o.jpg">
```

Comment:
Inline styling should be avoided due to issues with specificity (https://css-tricks.com/specifics-on-css-specificity/) and scale-ability.  Also, this links to an image resource that does not exist. Please correct. 🔨